src/pretix/base/metrics.py

```diff
@@ -0,0 +1,150 @@
+import redis
+
+ns_prefix = "pretix_metrics_"
+
+r = redis.Redis(host=redishost, port=redisport)
+
+
+class MetricsError(Exception):
+    def __init__(self, msg):
+        self.msg = msg
```

Comment:
That is the default constructor of ``Exception``, you can just omit it and do a ``class MetricsError(Exception):\n pass``